modules/oc-adm-by-example-content.adoc

```diff
@@ -340,31 +340,72 @@ Create a new project
 
 
 
+== oc adm node-image create
+Create an ISO image for booting the nodes to be added to the target cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Create the ISO image and download it in the current folder
+  oc adm node-image create
+  
+  # Use a different assets folder
+  oc adm node-image create --dir=/tmp/assets
+  
+  # Specify a custom image name
+  oc adm node-image create --o=my-node.iso
+  
+  # Create an ISO to add a single node without using the configuration file
+  oc adm node-image create --mac-address=00:d8:e7:c7:4b:bb
+  
+  # Create an ISO to add a single node with a root device hint and without
+  # using the configuration file
+  oc adm node-image create --mac-address=00:d8:e7:c7:4b:bb --root-device-hint=deviceName:/dev/sda
+----
+
+
+
+== oc adm node-image monitor
+Monitor new nodes being added to an OpenShift cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Monitor a single node being added to a cluster
+  oc adm node-image monitor --ip-addresses 192.168.111.83
+  
+  # Monitor multiple nodes being added to a cluster by separating each
+  IP address with a comma
```

Comment:
Note that I've submitted a fix for this line (should be commented out) here: https://github.com/openshift/oc/pull/1872